app/models/links.rb

```diff
@@ -0,0 +1,69 @@
+class Links
+  # Links available for indexed gems
+  LINKS = {
+    'home'     => 'homepage_uri',
+    'code'     => 'source_code_uri',
+    'docs'     => 'documentation_uri',
+    'wiki'     => 'wiki_uri',
+    'mail'     => 'mailing_list_uri',
+    'bugs'     => 'bug_tracker_uri',
+    'download' => 'download_uri'
+  }.freeze
+
+  # Links available for non-indexed gems
+  NON_INDEXED_LINKS = {
+    'docs'     => 'documentation_uri'
+  }.freeze
+
+  attr_accessor :rubygem, :version, :linkset
+
+  def initialize(rubygem = nil, version = nil)
```

Comment:
Why allow `nil` defaults when this class doesn't work with `nil` rubygem or version?